float64 x  # The x-coordinate of the point
float64 y  # The y-coordinate of the point
float64 vel_x  # The velocity in the x-direction
float64 vel_y  # The velocity in the y-direction
